packages/js/src/general/app.js

```diff
@@ -71,6 +69,7 @@ Menu.propTypes = {
  */
 const App = () => {
 	const notices = useMemo( getMigratingNoticeInfo, [] );
+	const resolvedNotices = select( STORE_NAME ).selectResolvedNotices();
```

Comment:
Not introduced in your PR, but we should rewrite `linkParams` in the same fashion 🙂